app/models/spree/image.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+require 'spree/core/s3_support'
```

Comment:
I think we might need this line in Bye Bye Spree...

Reply:
yesterday when trying bootboot I had to remove something like this to fix a spec, some error with loading this concern multiple times in Enterprise:
https://github.com/openfoodfoundation/openfoodnetwork/pull/6298/files#diff-31c82445b1c4ad30a20486262f1a746b1b2212baf265998d418d043ad005ce15L3

Maybe a mix of bye bye spree and rails 4.1 upgrade (I think auto class loading did change from 4.0 to 4.1).